src/store.js

```diff
@@ -20,7 +20,7 @@ import { ArrayIndexOf } from './util'
 import Formula from './formula'
 import { RDFArrayRemove } from './util'
 import Statement from './statement'
-import Node from './node'
```

Comment:
Hmm, this might just be my unfamiliarity with the codebase, but is this change intended/valid? Likewise for the same change in `variable.js`, `formula.js`, etc.

Reply:
Note sure about the implications, I'll revert them